src/main/asciidoc/reference-guide/tutorials/configdatabaseresource.adoc

```diff
@@ -0,0 +1,160 @@
+=== Configure a Database resource
+:hardbreaks:
+
+For Syncope 2.0.x
+
+* Introduction
+* Pre-requisites
+* Configure Database Table connector instance
+* Configure Database Table resource
+* Configure Resource Provision rules
+
+==== Introduction
+
+This tutorial shows all the steps needed to configure an external Database Table resource identified by the following connection parameters:
+
+[cols="2*"]
+|===
+|Host
+|localhost
+
+|Port
+|3306
+
+|User
+|root
+
+|User password
+|password
+
+|JDBC connection URL
+|jdbc:mysql://localhost:3306/demo
+
+|Database
+|demo
+
+|Table
+|test
+|===
+
+==== Pre-requisites
+
+. Download JDBC driver (MySQL JDBC driver in this case) and include it into your JEE container classpath.
+
+. Verify that the bundle exists or https://cwiki.apache.org/confluence/display/SYNCOPE/Install+connector+bundles[Install ConnId Database Table connector bundle]
```

Comment:
The docs in the wiki are old, and applies up to Syncope 1.2. You should link to other parts of this guide, instead.

This sentence becomes then:
```
Verify that the bundle exists or <<install-connector-bundles,install the ConnId Database Table connector bundle>>
```

as https://syncope.apache.org/docs/reference-guide.html#install-connector-bundles is the target link (the part after `#` is the actual link to use in the statement above).